cardano-cli/test/cardano-cli-golden/files/golden/help/allegra_transaction.cli

```diff
@@ -7,6 +7,7 @@ Usage: cardano-cli allegra transaction
                                          | submit
                                          | policyid
                                          | calculate-min-fee
+                                         | calculate-min-fee-2
```

Comment:
I'm not sure what's the use case of this command. If we mean to support offline transaction building, which now is:
```
build-raw -> calculate-min-fee -> sign
```
this command will be redundant because we would have to connect to the node to use it.
In the online process:
```
build -> sign
```
it's also not useful. 

More context (internal link): https://input-output-rnd.slack.com/archives/C057E9CJTPH/p1710941749044059?thread_ts=1710521123.756749&cid=C057E9CJTPH

@CarlosLopezDeLara what do you think?

Reply:
@gitmachtl @GeorgeFlerovsky @lehins 

I tried upgrading to the command to use the ledger's new and more accurate `calcMinTxFee`.

Because `calcMinTxFee` requires an additional `utxos` argument, I need to obtain this value from somewhere.   I took the simplest path which is the replicate code that produces this value from the `transaction build` which pulls in a *very* large number of dependencies.

This seems unlikely to be what we want.  I was wondering if you had some advice about how we intend the command to work given the additional `utxos` argument requirement.